src/test/java/com/microsoft/spring/data/gremlin/query/GremlinTemplateIT.java

```diff
@@ -357,5 +357,51 @@ public void testSaveEdge() {
         Assert.assertEquals(updatedRelationship.getName(), foundRelationship.getName());
         Assert.assertEquals(updatedRelationship.getLocation(), foundRelationship.getLocation());
     }
+
+    @Test
+    public void testVertexDeleteById() {
+        this.template.deleteById(this.person.getId(), Person.class);
+        this.template.insert(this.person);
+        this.template.deleteById(this.person0.getId(), Person.class);
+
+        Person foundPerson = this.template.findById(this.person.getId(), Person.class);
+        Assert.assertNotNull(foundPerson);
```

Comment:
Add any assertion failure message here?

Reply:
It is UT. I think it is clean enough here.